src/oskbd/macos.rs

```diff
@@ -219,10 +223,25 @@ impl KbdOut {
         self.write_key(key, KeyValue::Release)
     }
 
-    /// Send using C-S-u + <unicode hex number> + spc
     pub fn send_unicode(&mut self, c: char) -> Result<(), io::Error> {
-        log::error!("Unable to send unicode {c}, unsupported functionality");
-        todo!()
+        let event_source =
+            CGEventSource::new(CGEventSourceStateID::CombinedSessionState).map_err(|_| {
```

Comment:
I'm totally unfamiliar with macOS and core graphics. 

Is this something that would be more appropriate to initialize once on startup, or is creating a new `CGEventSource` and then dropping it at the end of the unicode call fine to do?

Reply:
Same here. I can give it a try to only initialize this once, intuitively it makes sense.

Reply:
Okay I played with this a bit but I have trouble getting it working. I tried adding a `Arc<Mutex<CGEventSource>>` field onto the `KbdOut` struct but this requires `CGEventSource` to implement `Send`, which it doesn't. I'm fairly new at Rust, do you have any pointers where I could go from here?

Reply:
If it's not Send then I think the code is fine as-is.